### DISCUSSION TITLE
Hyper Key

### DISCUSSION THREAD

**Discussion by melbaldove** [manna-harbour/miryoku#157] [Q&A] [answered]
What are suggestions for binding the Hyper key?

**Comment by manna-harbour** [comment, marked as answer]
What's your use case?

I don't think Hyper is usually bound by itself as a mod. It's usually used to make unique keybindings for custom shortcuts on the host, so you'd add bindings for each individual shortcut key combination. But for that purpose I just use Meta and pick a key combination not already used by the window manager. Hyper can't be combined with other mods, so it's not as flexible.

To get Hyper from stock Miryoku you can just hold all the mods. If you want to add it as a mod you can add it anywhere as a mod tap as you won't be able to combine with other mods anyway. Most mod tap locations are already in use, but you could replace AltGr if you don't need it, or add it to the inner index column.

**Comment by melbaldove** [comment]
Actually I think its pretty okay to just hold down all the mods since they
are nicely lined up on the homerow. Thanks
